### ISSUE TITLE
Merge of #61 breaks existing projects

### ISSUE THREAD

**Issue by TobiX** [stevespringett/dependency-check-sonar-plugin#82]
Hi,

I have existing projects where the Dependency Check report file is not an "input file" for SonarQube. Since #61 assumes the XML report file is an input file, no issues are found in all those projects any more. This would be a non-trivial change to ~200 internal projects, so I would really appreciate if a different solution could be found.

**Comment by Reamer** [comment]
@pethers 
What's your opinion to this issue? In my merge-Request #79 , I removed this commit, because I recognize this problem, too.
In my opinion the xml report file shouldn't be a part of the sonar input files. The report file is more or less not readable and not the root-reason of the dependency.

I tried a little bit to work around this problem #55 and this is a first version https://github.com/Reamer/dependency-check-sonar-plugin/commit/c9a3abdf6ad46b7246db08a3a6bedbf7d89fc9e4
With this commit the plugin tries to find a "Dependency-Reason". In a classic maven project it's a "pom.xml" (which is part of sonar-input files). The Issues are linked to the pom.xml. If we are lucky and the dependency is not a transitive dependency, it's also possible to link to a specific line.
